source/ui/BioXAS/BioXASZebraLogicBlockInputInvertedView.cpp

```diff
@@ -0,0 +1,77 @@
+#include "BioXASZebraLogicBlockInputInvertedView.h"
+
+BioXASZebraLogicBlockInputInvertedView::BioXASZebraLogicBlockInputInvertedView(BioXASZebraLogicBlockInput *control, QWidget *parent) :
```

Comment:
Okay! I have some questions about how best to go about this, when you have a second @dretrex.